.github/workflows/ci.yml

```diff
@@ -0,0 +1,118 @@
+---
+name: OpenWISP Monitoring CI Build
+
+on:
+  push:
+    branches:
+      - master
+      - dev
+  pull_request:
+    branches:
+      - master
+      - dev
+
+jobs:
+  build:
+    name: Python==${{ matrix.python-version }} | ${{ matrix.django-version }}
+    runs-on: ubuntu-20.04
+
+    services:
+      redis:
+        image: redis
+        ports:
+          - 6379:6379
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - 3.6
+          - 3.7
+        django-version:
+          - django~=2.2
```

Comment:
let's drop django 2.2

Reply:
@nemesisdesign @atb00ker there is an issue opened for that. Should I link that issue too here, or instead make a new pull request

Reply:
@nemesisdesign @devkapilbansal I would prefer a separate commit to drop support for Django 2.2. It makes it easy to track changes from git log.